A6-client-retries.md

```diff
@@ -106,9 +106,9 @@ gRPC's call deadline applies across all attempts for a given RPC. For example, i
 
 ##### Exponential Backoff
 
-The `initialBackoff`, `maxBackoff`, and `backoffMultiplier` parameters determine the randomized delay before retry attempts.
+The `initialBackoff`, `maxBackoff`, and `backoffMultiplier` parameters tune the exponential delay before retry attempts.
 
-The initial retry attempt will occur at `random(0, initialBackoff)`. In general, the `n`-th attempt will occur at `random(0, min(initialBackoff*backoffMultiplier**(n-1), maxBackoff))`.
+The initial retry attempt will occur after `initialBackoff`.  After that, the `n`-th attempt will occur after `min(initialBackoff*backoffMultiplier**(n-1), maxBackoff) * random(0.8, 1.2))`.
```

Comment:
The initial attempt should also have the jitter. Ditto in the schema documentation.

Reply:
Good point!  Fixed.

It looks like C-core's backoff implementation isn't actually doing this, so I'll fix that too. :)